src/content/app/species-selector/components/popular-species-list/PopularSpeciesList.tsx

```diff
@@ -43,8 +44,10 @@ const PopularSpeciesList = () => {
 
   const onPopularSpeciesButtonClick = (species: PopularSpecies) => {
     dispatch(setPopularSpecies(species));
-    dispatch(setModalView('popular-species-genomes'));
     trackPopularSpeciesClick(species);
+    navigate(
+      `/species-selector/search?species_taxonomy_id=${species.species_taxonomy_id}`
```

Comment:
We probably want to move the generation of these urls into the urlHelper file.

Reply:
✅